content/docs/_index.md

```diff
@@ -41,6 +41,14 @@ This is a community driven project and our roadmap is publicly available on our
 
 ## Helpful Videos
 
+[Intro to Cloud Native Buildpacks (KubeCon NA 2019)](https://www.youtube.com/watch?v=SK6e_ZatOaw)
+
+  - Terence and Emily discuss how buildpacks work, why the project was created, and give some demos
+
+[Deep Dive: Cloud Native Buildpacks (KubeCon NA 2019)](https://www.youtube.com/watch?v=j9Ak5YLrihU)
+
+  - Stephen and Joe describe why you need a buildpack, how to create one, and what you'll gain from using them
+
 [How Pivotal Cloud Foundry Buildpacks Work](https://www.youtube.com/watch?v=1JKLiPBmlIc)
```

Comment:
Minor: We should probably remove this video as it's discussing Buildpack v2 concepts.

![image](https://user-images.githubusercontent.com/475559/71212036-1ad6ee00-2276-11ea-8a9d-1b99c894ca6c.png)